wolfssl/internal.h

```diff
@@ -2891,6 +2891,39 @@ typedef enum {
 #endif
 } TLSX_Type;
 
+/* Same list a the above enum TLSX_Type */
```

Comment:
I'd prefer if you could just make the above `TLSX_Type` types always available and use those directly.

Reply:
Remove these... _DEFINE types (not used now)